node_modules/oae-email/lib/api.js

```diff
@@ -331,10 +328,7 @@ var sendEmail = module.exports.sendEmail = function(templateModule, templateId,
                 'err': txtErr,
                 'templateModule': templateModule,
                 'templateId': templateId,
-                'toUser': {
-                    'id': toUser.id,
-                    'locale': toUser.locale
```

Comment:
Do we not need to pass in the locale anymore to calculate the times in the email?